### ISSUE TITLE
Unable to install packages - pathspec 'master' did not match any file(s) known to git

### ISSUE THREAD

**Issue by standaloneSA** [jsonnet-bundler/jsonnet-bundler#47]
On a CentOS 7 machine with the most recent version of jb installed through 'go get  github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb', with git version 2.16.5

Trying to install kube-prometheus in this case, via 

\#jb install github.com/coreos/prometheus-operator/contrib/kube-prometheus/jsonnet/kube-prometheus

I get 

```
 jb install github.com/coreos/prometheus-operator/contrib/kube-prometheus/jsonnet/kube-prometheus
GET https://github.com/coreos/prometheus-operator/archive/master.tar.gz 200
archive install failed: etag value "W/"1b6e9e79ab596d977e77fd7ca473b19cd12a1c594f6d14360e136666d32e29d6"" does not look like a SHA1
retrying with git...
Initialized empty Git repository in /root/code/my-kube-prom/vendor/.tmp/jsonnetpkg-kube-prometheus-master316687774/.git/
remote: Enumerating objects: 3075, done.
remote: Counting objects: 100% (3075/3075), done.
remote: Compressing objects: 100% (2379/2379), done.
remote: Total 3075 (delta 846), reused 1528 (delta 495), pack-reused 0
Receiving objects: 100% (3075/3075), 7.32 MiB | 13.21 MiB/s, done.
Resolving deltas: 100% (846/846), done.
From https://github.com/coreos/prometheus-operator
 * branch            master     -> FETCH_HEAD
error: pathspec 'master' did not match any file(s) known to git.
jb: error: failed to install: failed to install package: exit status 1
```

Same error with every other bundle I've tried installing, where the pathspec did not match any files known to git. 

Is there something I'm doing wrong, in this case?

**Comment by gerrit8143** [comment]
looks like the etag value is SHA256 and not SHA1

**Comment by gerrit8143** [comment]
@standaloneSA the 0.1.0 release works fine

**Comment by bgagnon** [comment]
Indeed, the archive download optimization is currently broken as described in #46. However, the fallback to `git` should still function.

So this bug is still unaccounted for:

```
error: pathspec 'master' did not match any file(s) known to git.
jb: error: failed to install: failed to install package: exit status 1
```

Any idea what is causing it, @gerrit8143?

**Comment by bgagnon** [comment]
The etag problem was fixed in #49 but the other part of this issue still has no explanation or fix.

**Comment by sh0rez** [comment]
I tried to reproduce today, no success. As there hasn't been any activity on this issue for a longer time, I'm closing it. Feel free to open a new one if the problem persists!
